src/sage/rings/lazy_series.py

```diff
@@ -171,30 +171,31 @@
     ....:         yield n
     ....:         n += 1
 
-    sage: L.<z> = LazyLaurentSeriesRing(GF(2))
-    sage: check(L, lambda n: n, valuation=-5)
-    sage: check(L, gen(), valuation=-5)
+    sage: L.<z> = LazyLaurentSeriesRing(GF(2))                                          # optional - sage.rings.finite_rings
+    sage: check(L, lambda n: n, valuation=-5)                                           # optional - sage.rings.finite_rings
+    sage: check(L, gen(), valuation=-5)                                                 # optional - sage.rings.finite_rings
 
     sage: L = LazyDirichletSeriesRing(QQbar, "s")
     sage: check(L, lambda n: n, valuation=2)
     sage: check(L, gen(), valuation=2)
 
```

Comment:
`sage.rings.number_field?

By the way, this being singular is not consistent with `sage.rings.finite_rings` being plural...

Reply:
Yes, I'm also mildly annoyed by this inconsistency

Reply:
Done in b9bf5bbe02